src/ScriptBuilder/Durations/DurationDefinitionReader.cs

```diff
@@ -4,22 +4,21 @@
     using System.Linq;
     using Mono.Cecil;
 
-    static class MeterDefinitionReader
+    static class DurationDefinitionReader
     {
-        public static bool TryGetMeterDefinition(TypeDefinition type, out List<MeterDefinition> definitions)
+        public static bool TryGetDurationDefinition(TypeDefinition type, out List<DurationDefinition> definitions)
         {
-            definitions = new List<MeterDefinition>();
-            if (type.BaseType != null && type.BaseType.FullName == "NServiceBus.Metrics.MetricBuilder")
+            definitions = new List<DurationDefinition>();
+            if (type.BaseType != null && type.BaseType.FullName == "DurationProbeBuilder")
             {
-                var attributes = type.Fields
-                    .Select(f => f.GetSingleAttribute("NServiceBus.Metrics.MeterAttribute"))
-                    .Where(c => c != null);
+                var attributes = type.CustomAttributes.Where(a => a.AttributeType.FullName == "ProbePropertiesAttribute");
```

Comment:
Yes, that's correct https://github.com/Particular/NServiceBus.Metrics/blob/develop/src/NServiceBus.Metrics/ProbeBuilders/ProbePropertiesAttribute.cs